app/assets/javascripts/discourse/app/components/topic-list/item/likes-cell.gjs

```diff
@@ -0,0 +1,18 @@
+import { gt } from "truth-helpers";
+import number from "discourse/helpers/number";
+import icon from "discourse-common/helpers/d-icon";
+
+const LikesCell = <template>
+  {{#if @showLikes}}
```

Comment:
Adding these kind of conditionals to multiple cells on every single row feels a little wasteful. Could we instead add some kind of `shouldRender()` function which is evaluated when building the DAG? That way, we only need to evaluate/autotrack the condition once, rather than once-per-row.

IIRC that's the approach @megothss has taken on the post menu. It kinda mimics our PluginOutlet API.